subsys/mgmt/Kconfig.mcumgr

```diff
@@ -10,8 +10,13 @@ config MCUMGR
 
 if MCUMGR
 
-menu "Command handlers"
+config MGMT_CBORATTR_MAX_SIZE
+  int "The maximum size of a CBOR attribute during decoding"
+	default 512
+	help
```

Comment:
The indentation seems to be off here for the last two lines

Reply:
fixed